src/backend/renderer/multigpu/gbm.rs

```diff
@@ -114,17 +121,23 @@ impl<R> GbmGlesBackend<R> {
         }
     }
 
+    /// Sets the default flags to use for allocating buffers via the [`GbmAllocator`]
+    /// provided by these backends devices.
+    ///
+    /// Only affects nodes added via [`add_node`] *after* calling this method.
+    pub fn set_allocator_flags(&mut self, flags: GbmBufferFlags) {
+        self.allocator_flags = flags;
+    }
```

Comment:
This looks more like something to set during initialization. Maybe this shoule be another constructor like `GbmGlesBackend::with_allocator_flags`?
(Might be annoying to write as it would require to also support combinations with `with_factory` and `with_context_priority` or switch to a builder)

Reply:
Yes this was mostly to avoid all the permutations of existing constructors. If you have a better idea feel free, but a builder pattern also isn't exactly easy, given that with_factory and with_context_priority exclude each other.

Reply:
Okay, also not a big deal and it is actually documented.